saas_app/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,46 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.231.6/containers/docker-existing-docker-compose
+// If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
+{
+	"name": "Existing Docker Compose (Extend)",
+
+	// Update the 'dockerComposeFile' list if you have more compose files or use different names.
+	// The .devcontainer/docker-compose.yml file contains any overrides you need/want to make.
+	"dockerComposeFile": [
+		"../docker-compose.yml",
```

Comment:
I recommend only using one docker-compose.yml file in this case. I think the sample set up intends for you to use a docker-compose.yml to provision the production database as well. In our case we would deploy that not in docker but in AWS RDS. So the best change would be to keep the content of the "../docker-compose.yml", paste that into "docker-compose.yml" and keep only that file in the .devcontainer directory.

Reply:
sigh.... One day I will get those devcontainers right. :-(
I made the change as requested.

Reply:
Thank you!